ckanext/aberdeen/actions.py

```diff
@@ -179,3 +182,69 @@ def group_purge(context, data_dict):
 
 def organization_purge(context, data_dict):
     return _group_or_org_purge(context, data_dict, is_org=True)
+
+
+@toolkit.side_effect_free
+def inactive_users(context, data_dict):
+    '''Returns a list of users that have been inactive for the last year'''
+
+    if not authz.is_sysadmin(toolkit.c.user):
+        toolkit.abort(403, _('You are not authorized to access this list'))
+
+    user_list = ckan.logic.get_action('user_list')(context, data_dict)
+    inactive_users = []
+    threads = []
+
+    # Limit the user activity results to the most recent event
+    data_dict['limit'] = 1
+
+    # Use threading to speed up the inactive user collection
+    def user_activity_threads(user, inactive_users):
+        data_dict['id'] = user['id']
+        user_info = ckan.logic.get_action(
+            'user_activity_list')(context, data_dict)
+
+        if not user_info:
+            return
+
+        timestamp = datetime.strptime(
+            user_info[0]['timestamp'], '%Y-%m-%dT%H:%M:%S.%f')
+
+        if timestamp < datetime.today() - timedelta(days=365):
```

Comment:
So as I understand this will count as inactive everybody that was inactive for 1 year. Can you make it configurable and default to 365? Eg `ckanext.aberdeen.inactive_for=30`

Reply:
Also, you will need to update README

Reply:
Yeah, great idea

Reply:
I added configurability for days

TODO: Update README

Reply:
I made the API call variable `days`, `http://ckan-dev:5000/api/action/inactive_users?days=180` for example. I still need to implement through the config file.

Reply:
I changed the `days` variable to `days_inactive`, and I added a check to see if it's declared in the config file under `ckanext.aberdeen.days_inactive`.